5.6/apache/Dockerfile

```diff
@@ -24,6 +24,27 @@ RUN apt-get update \
     && apt-get autoremove -y \
     && rm -r /var/lib/apt/lists/*
 
+# Add Yii2 config
+RUN { \
+        echo '<Directory /var/www/html/web>'; \
+        echo '  Require all granted'; \
+        echo '  # use mod_rewrite for pretty URL support'; \
+        echo '  RewriteEngine on'; \
+        echo '  # If a directory or a file exists, use the request directly'; \
+        echo '  RewriteCond %{REQUEST_FILENAME} !-f'; \
+        echo '  RewriteCond %{REQUEST_FILENAME} !-d'; \
+        echo '  # Exclude static file types'; \
+        echo '  RewriteCond %{REQUEST_URI} !\.(css|gif|ico|jpg|js|map|png|svg|ttf|woff|woff2)$'; \
+        echo '  # Otherwise forward the request to index.php'; \
+        echo '  RewriteRule . index.php'; \
+        echo '</Directory>'; \
+    } | tee /etc/apache2/conf-available/yii2-php.conf \
```

Comment:
Yes, I was also considering that. I'm not sure, why the use this approach at docker/php. But I will change it.